packages/react-core/src/components/NotificationDrawer/__tests__/__snapshots__/NotificationDrawer.test.tsx.snap

```diff
@@ -1,9 +1,9 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`className is added to the root element 1`] = `"pf-c-notification-drawer extra-class"`;
+exports[`className is added to the root element 1`] = `"extra-class"`;
 
 exports[`renders with PatternFly Core styles 1`] = `
-<div
-  className="pf-c-notification-drawer"
+<NotificationDrawerBase
+  innerRef={null}
```

Comment:
I'm also unsure if this test is still really testing what it's intending to now.